src/models/tiles/geometry/geometry-content.ts

```diff
@@ -364,16 +357,6 @@ export const GeometryContentModel = GeometryBaseContentModel
     let batchChanges: string[] = [];
 
     function handleWillApplyChange(board: JXG.Board | string, change: JXGChange) {
-      const op = change.operation.toLowerCase();
-      const target = change.target.toLowerCase();
-
-      // TODO Remove this or change metadata to tilecontainer or something
-      if ((op === "update") && (target === "metadata")) {
-        const props = change?.properties as JXGProperties | undefined;
-        if (props?.title) {
-          self.setTitle(props.title);
-        }
-      }
```

Comment:
There is some chance that there are legacy documents that recorded their title in the geometry change events. If that was the case then these geometry tiles would lose their title if they were loaded. However, it seems these legacy Geometry tiles also stored their title in the content model directly. And that is handled by the `tileSnapshotPreprocessor` in `geometry-registration.ts`.  Because of this it seems safe to remove this handling of these title change events.